api/v1alpha1/neighbor.go

```diff
@@ -0,0 +1,70 @@
+// SPDX-License-Identifier:Apache-2.0
+
+package v1alpha1
+
+import metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+type DynamicASNMode string
+
+const (
+	InternalASNMode DynamicASNMode = "internal"
+	ExternalASNMode DynamicASNMode = "external"
+)
+
+// Neighbor represents a BGP Neighbor we want FRR to connect to.
+type Neighbor struct {
+	// ASN is the AS number to use for the local end of the session.
+	// ASN and DynamicASN are mutually exclusive and one of them must be specified.
```

Comment:
DynamicASN is not here (so ASN shouldn't be +optional)

Reply:
done